src/charm.py

```diff
@@ -133,7 +149,25 @@ def _reconcile(self, _: ops.EventBase) -> None:
             self.container.start("exporter")
         else:
             self.container.stop("exporter")
-        self.unit.status = ops.ActiveStatus()
+        deadline = time.time() + 300
```

Comment:
IINM, there is already an "alive" check in the service, that will restart the workload container in less than 300s.

Maybe also a case for [defer/notify](https://juju.is/docs/sdk/how-and-when-to-defer-events) as the blocking is very long?

Reply:
I explained the reason a little bit in here https://github.com/canonical/penpot-operator/pull/9#discussion_r1726392199

Typically, this would take less than a minute; the 5-minute timeout is just a safe margin. Since this feature is not very essential to the Penpot charm, I don't want to invest too much effort into implementing it. At that point, it would be more economical to remove this feature.

Reply:
Sure, let then leave the active wait.

However, I think the 300s is not really true as I think the next check is in the code:
```
                "backend-ready": {
                    "override": "replace",
                    "level": "alive",
                    "period": "30s",
```
with the default threshold of 30 plus 10 seconds of timeout this will not be more than (30+10)*3 = 120s. The pod will be killed in 120s, so I think that the check or the maximum time to wait should be changed.

Reply:
Yes, since there are no actual consequences of the waiting timeout now, as it will just go into the block state, instead of error state, I think I can decrease the timeout value to 120 seconds here. Thanks!